deployments/helm/values.yaml

```diff
@@ -67,6 +67,8 @@ securityContext: {}
   # runAsNonRoot: true
   # runAsUser: 1000
 
+priorityClassName: ""
```

Comment:
Do you mind if we add an example here of what this could be? E.g. `system-node-critical`?